Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -0,0 +1,536 @@
+import json
+
+import dateparser
+import demistomock as demisto
+import requests
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+
+# IMPORTS
+
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# CONSTANTS
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+MAX_INCIDENTS_TO_FETCH = 50
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def alert_list(self, url_suffix, request_data):
+        data = self._http_request(
+            method='GET',
+            url_suffix=url_suffix,
+            json_data=request_data
+        )
+        return data
+
+    def alert_dismiss_bulk(self, request_data):
+        data = self._http_request(
+            method='POST',
+            url_suffix='/alerts/dismiss_bulk/',
+            json_data=request_data
+        )
+        return data
+
+    def alert_resolve_bulk(self, request_data):
+        data = self._http_request(
+            method='POST',
+            url_suffix='/alerts/resolve/',
+            json_data=request_data
+        )
+        return data
+
+    def activities_list(self, url_suffix, request_data):
+        data = self._http_request(
+            method='GET',
+            url_suffix=url_suffix,
+            json_data=request_data
+        )
+        return data
+
+    def users_accounts_list(self, request_data):
+        data = self._http_request(
+            method='GET',
+            url_suffix='/entities/',
+            json_data=request_data
+        )
+        return data
+
+    def files_list(self, url_suffix, request_data):
+        data = self._http_request(
+            method='GET',
+            url_suffix=url_suffix,
+            json_data=request_data
+        )
+        return data
+
+    def list_incidents(self, filters, limit):
+        """
+        returns dummy incident data, just for the example.
+        """
+        return self._http_request(
+            method='POST',
+            url_suffix='/alerts/',
+            json_data={
+                'filters': filters,
+                'limit': limit
+            }
+        )
+
+
+def arg_to_timestamp(arg):
+    if isinstance(arg, str) and arg.isdigit():
+        return int(arg)
+    if isinstance(arg, str):
+        date = dateparser.parse(arg, settings={'TIMEZONE': 'UTC'})
```

Comment:
Are you sure that `'TIMEZONE': 'UTC'` will work for  customers that aren't in UTC?